### DISCUSSION TITLE
Autofill email for user from password manager

### DISCUSSION THREAD

**Discussion by nebula2** [dotnet/maui#13312] [Q&A]
How to make it so that an entry suggests saved credentials from password managers?

I have a login form where you just give email. I have set Entry's Keyboard to Email. The Layout is correct but it does not suggest anything :/

**Comment by fredericDelaporte** [comment]
It does not seem to have any support for this out of the box in MAUI as of now.

An old issue asking to get that feature back from Xamarin seems stale. See CommunityToolkit/Maui#373.

There is also a discussion stating that using `AutomationId` could handle it, but it seems they are not supposed to be meant for that. See #18966.
